### PR TITLE
Fixed data not to have stats property.

### PR DESCRIPTION
**Pull request by I-MirzaBaig**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4682"></a>

### CODE REVIEW

File: data/I-MirzaBaig.json [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
-      "url": "https://hashnode.com/@AmaanMirza",
-      "icon": "FaHasnode"
-    },
-    {
-      "name": "Follow me on Showwcase",
-      "url": "https://www.showwcase.com/amaanmiirza",
-      "icon": "FaShowwcase"
-    }
-  ]
+< main
```
Comment (priyankarpal) [inline data/I-MirzaBaig.json:2]: Delete this line.
```suggestion

```

File: data/I-MirzaBaig.json [hunk 2]
```diff
@@ -1,46 +1,47 @@
...
+    "name": "Mirza Baig",
+    "type": "personal",
+    "bio": "Open Source **DevRel** enthusiast | **Devops** enthusiast | **Travel** enthusaist" ,
+    "avatar": "https://github.com/I-MirzaBaig.png",
```
Comment (priyankarpal) [inline data/I-MirzaBaig.json:6]: Currently, you don't need to add an avatar code. 
```suggestion
   
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/I-MirzaBaig.json [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
+      },
+      {
+        "name": "LinkedIn: Let's connect",
+        "url": "https://www.linkedin.com/in/amaan-mirza-b10b06208/",
+        "icon": "FaLinkedin"
+      },
+      {
+        "name": "Follow me on Hashnode", 
+        "url": "https://hashnode.com/@AmaanMirza", 
+        "icon": "FaHasnode"
```
Comment (swagataroy30) [inline data/I-MirzaBaig.json:39]:
```suggestion
        "icon": "SiHashnode"
```

**Review by swagataroy30 — COMMENTED** [review]
There is a typo and also `FaHashnode` is not available. can you please change it to `SiHashnode`. Thank you!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**